Taskfile.yml

```diff
@@ -89,21 +89,8 @@ tasks:
   build-docs-site:
     dir: docs/hugo
     cmds:
-      - npm install -g postcss postcss-cli autoprefixer
       - hugo
-    # - htmltest
-    # disabled pending: https://github.com/wjdp/htmltest/issues/45
-    # , or us moving to a no-directory URL
-    env:
-      NODE_PATH:
-        sh: npm root -g
-
-  check-docs-site-links:
-    desc: Verify documentation site links
-    dir: docs/hugo
-    cmds:
-      # Excluding './content/reference/*' path as it contains all auto-generated API docs.
-      - find . -type f -name "*.md" -not -path "./content/reference/*" -not -path "./node_modules/*" -print0 | xargs -0 -n1 markdown-link-check -c link-checker.json
+      - htmltest
```

Comment:
how does this work? do we not require to provide any path?

Reply:
The task runs in `docs/hugo` (see line 90) and there's a `.htmltest.yml` file in that folder that has the rest of the configuration.